NeuroID/src/main/java/com/neuroid/tracker/NeuroIDPublic.kt

```diff
@@ -0,0 +1,190 @@
+package com.neuroid.tracker
+
+import android.app.Activity
+import androidx.annotation.VisibleForTesting
+import com.neuroid.tracker.models.SessionStartResult
+
+interface NeuroIDPublic {
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("Replaced with getClientID", ReplaceWith("getClientID()"))
+    fun getClientId():String
+
+    /**
+     * Return the currently set client id (use this one).
+     */
+    fun getClientID():String
+
+    /**
+     * Update the test event sender endpoint to a new collection endpoint. This will generally be
+     * set to the mock collection endpoint. The remote config endpoint will be updated to point to
+     * the dev remote config endpoint.
+     */
+    fun setTestURL(newEndpoint: String)
+
+    /**
+     * Set a dev testing URL for testing.
+     */
+    @VisibleForTesting
+    fun setTestingNeuroIDDevURL()
```

Comment:
Since this is used only for internal testing does this need to be in the public interface? I suppose we could use this for flexibility when we are testing but just wanted to check if that's the reason.

Reply:
I would have liked to have made this internal however the sandbox sample app that is bundled with the SDK uses this as well. so this needs to be publicly accessible. Will look into fixing this in the future (move that test into a unit test).